### PR TITLE
Conclusion

### PR DESCRIPTION
**Pull request by Ivikhostrup**

### CODE REVIEW

File: report/src/conclusion/conclusion.tex [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Of these technologies, Bluetooth Low Energy was found to be the best candidate. 
+Three techniques for presence detection using BLE were presented and analyzed in Chapter \ref{sec:presence_detection_using_ble}. 
+It was found that fingerprinting using k nearest-neighbors would be a suitable candidate for presence detection.
+
+Subsequently, we conducted an experimental evaluation to determine if the selected technique could effectively contribute to a system aimed at resolving the stakeholder's problems. 
+The development of a mobile application, implemented in C\#, was created to facilitate these experiments. 
+The results of the experiments indicated that our approach could achieve satisfactory accuracy in detecting whether a person was present within the room where the experiments were carried out.
+
+In conclusion, the findings of our investigation suggest that presence detection technology, particularly Bluetooth Low Energy, can be reliably utilized to automatically detect and indicate the status of a meeting, therefore minimizing the reliance on user interaction.
```
Comment (Pattrigue) [inline report/src/conclusion/conclusion.tex:13]:
```suggestion
In conclusion, the findings of our investigation suggest that presence detection technology, particularly Bluetooth Low Energy, can be reliably utilized to automatically detect and indicate the status of a meeting, therefore minimizing the reliance on user interaction.
```
This research indicates that presence detection systems, particularly those based on BLE, can effectively automate the tracking of meeting room occupancy. By reducing reliance on human intervention, these systems could revolutionize the management of meeting spaces, leading to more efficient use and, consequently, a potential reduction in maintenance costs.

File: report/src/conclusion/conclusion.tex [hunk 2]
```diff
@@ -0,0 +1,13 @@
+\chapter{Conclusion}
+This report examines a stakeholder's problems regarding the management of meetings. 
+By performing a usability evaluation of a developed prototype, it is concluded that a solution to their problem might involve a presence detection system to establish whether the stakeholder's meeting rooms are occupied.
```
Comment (Pattrigue) [inline report/src/conclusion/conclusion.tex:3]:
```suggestion
Summarizing this investigation, the report examines the complexities encountered in managing meeting spaces. 
Through a usability evaluation of a devised prototype, the research concludes that a viable solution to the identified problem could be the deployment of a presence detection system to determine the occupancy status of the stakeholder's meeting rooms.
```

File: report/src/conclusion/conclusion.tex [hunk 3]
```diff
@@ -0,0 +1,13 @@
+\chapter{Conclusion}
+This report examines a stakeholder's problems regarding the management of meetings. 
+By performing a usability evaluation of a developed prototype, it is concluded that a solution to their problem might involve a presence detection system to establish whether the stakeholder's meeting rooms are occupied. 
+Chapter \ref{chap:presence_intro} describes and evaluates technologies appropriate for presence detection, guided by a set of criteria considered necessary to address the stakeholder's problem. 
+Of these technologies, Bluetooth Low Energy was found to be the best candidate. 
+Three techniques for presence detection using BLE were presented and analyzed in Chapter \ref{sec:presence_detection_using_ble}. 
+It was found that fingerprinting using k nearest-neighbors would be a suitable candidate for presence detection.
```
Comment (Pattrigue) [inline report/src/conclusion/conclusion.tex:7]:
```suggestion
Through an exploration of technologies suitable for presence detection, an extensive evaluation was conducted to address the stakeholder's problem (see Chapter \ref{chap:presence_intro}). 
After careful consideration, Bluetooth Low Energy (BLE) emerged as the most promising choice. 
Building upon this foundation, Chapter \ref{sec:presence_detection_using_ble} delves into three distinct techniques for presence detection utilizing BLE.
Following a detailed assessment, the fingerprinting technique, using the k-nearest neighbors approach, emerged as a suitable candidate for presence detection.```
I don't like the way this is phrased. It feels like a dull walkthrough and doesn't flow well. I suggest we fix this by adding some coherence - see my suggestion - using words such as "following", "building upon", and so on.

Comment (Pattrigue) [reply report/src/conclusion/conclusion.tex:7]: I don't like the way this is phrased. It feels like a dull walkthrough and doesn't flow well. I suggest we fix this by adding some coherence - see my suggestion - using words such as "following", "building upon", and so on.

File: report/src/conclusion/conclusion.tex [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+Subsequently, we conducted an experimental evaluation to determine if the selected technique could effectively contribute to a system aimed at resolving the stakeholder's problems. 
+The development of a mobile application, implemented in C\#, was created to facilitate these experiments. 
+The results of the experiments indicated that our approach could achieve satisfactory accuracy in detecting whether a person was present within the room where the experiments were carried out.
```
Comment (Pattrigue) [inline report/src/conclusion/conclusion.tex:11]:
```suggestion
After conducting an experimental evaluation to assess the effectiveness of the selected technique in addressing the stakeholder's problems, we proceeded with the development of a mobile application implemented in C#. 
This application served as a facilitator for the experiments conducted. 
The obtained results from these experiments demonstrated that our approach yielded satisfactory accuracy in detecting the presence of individuals within the experimental room.
```

Comment (Pattrigue) [reply report/src/conclusion/conclusion.tex:11]: I think this needs to be rephrased as well, it reads quite poorly. For example, this sentence:
"The development of a mobile application, implemented in C\#, was created to facilitate these experiments." doesn't make any sense. Removing the interposed sentence leaves us with:
"The development of a mobile application was created to facilitate these experiments.", implying that we created a development. 

Also, when talking about the results of the experiment, should we reference back to that chapter? You did that in several other cases, but why not with the experimental results too?

File: report/src/conclusion/conclusion.tex [hunk 5]
```diff
@@ -0,0 +1,13 @@
+\chapter{Conclusion}
+Summarizing this investigation, the report examines the complexities encountered in managing meeting spaces. 
+Through a usability evaluation of a devised prototype, it was concluded that a viable solution to the identified problem could be the deployment of a presence detection system, to determine the occupancy status of the stakeholder's meeting rooms.
+By exploring technologies suitable for presence detection, an extensive analysis was conducted to select an appropriate technology for such a system (see Chapter \ref{chap:presence_intro}). 
+After careful consideration, Bluetooth Low Energy (BLE) emerged as the most promising technology. 
+Chapter \ref{sec:presence_detection_using_ble} details three distinct techniques for presence detection utilizing BLE.
```
Comment (Pattrigue) [inline report/src/conclusion/conclusion.tex:6]: You just copied my suggestion and changed it to a summary. Why? This is a conclusion, not a summary.
```suggestion
Building upon this foundation, Chapter \ref{sec:presence_detection_using_ble} delves into three distinct techniques for presence detection utilizing BLE.
```

File: report/src/conclusion/conclusion.tex [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+Following a detailed assessment, the fingerprinting technique, using the k-nearest neighbors approach, emerged as a suitable candidate for presence detection.
+Having established that this technique was a suitable candidate, we proceeded with the development of a mobile application implemented in C#. 
+This application served as a facilitator for experiments used to establish whether the approach was suitable for the aforementioned system.
```
Comment (Pattrigue) [inline report/src/conclusion/conclusion.tex:9]:
```suggestion
This application was utilized in carrying out experiments to assess if the approach was fitting for the system in question.
```

File: report/src/conclusion/conclusion.tex [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+After careful consideration, Bluetooth Low Energy (BLE) emerged as the most promising technology. 
+Chapter \ref{sec:presence_detection_using_ble} details three distinct techniques for presence detection utilizing BLE.
+Following a detailed assessment, the fingerprinting technique, using the k-nearest neighbors approach, emerged as a suitable candidate for presence detection.
+Having established that this technique was a suitable candidate, we proceeded with the development of a mobile application implemented in C#. 
+This application served as a facilitator for experiments used to establish whether the approach was suitable for the aforementioned system. 
+The obtained results from these experiments demonstrated that our approach yielded satisfactory precision in detecting the presence of individuals within the experimental room.
+
+This indicates that presence detection systems using BLE and fingerprinting can effectively automate the tracking of meeting room occupancy.
+By minimizing the dependence on human interaction, these systems could enhance the management of meeting rooms, leading to more efficient use and, consequently, a potential reduction in administrative costs.
```
Comment (Pattrigue) [inline report/src/conclusion/conclusion.tex:13]:
```suggestion
By minimizing the dependence on human interaction, these systems could enhance the management of meeting rooms, leading to more efficient use and a potential reduction in administrative costs.
```

File: report/src/conclusion/conclusion.tex (hunk 6 above)
Comment (chhoumann) [reply report/src/conclusion/conclusion.tex:9]:
```suggestion
This application was used to carry out experiments to assess the approach's suitability for the system in question.
```

**Review by chhoumann — APPROVED** [review]

**Pull request merged**